src/app/App.js

```diff
@@ -1,6 +1,9 @@
+import createLogo from './view/createLogo';
+
 const App = ({ title, options }) => {
-  const quizTitle = document.createElement('h1');
-  quizTitle.textContent = document.querySelector('#hpquiz-app').append(title);
+  // const quizTitle = document.createElement('h1');
+  // quizTitle.textContent = document.querySelector('#hpquiz-app').append(title);
+  createLogo();
```

Comment:
merge newest development into your branch and in view.js uncomment line number 47 and there you can activate your createLogo;

`this.render('.header__game-logo', createlogo);`